# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/xdr/bug-1604747.js
# Commit: d4359a64d30f
# Full Hash: d4359a64d30fe311f0d3aff7b0e50e57874a0cd2
# Author: caroline <cullen.caroline@gmail.com>
# Date: 2019-12-20 21:54:48
# Description:
#   Bug 1604747 - Error if attempt at encoding and instantiated module. r=iain
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D57900
# ==============================================================================

diff -r 0480be98e2a9 -r d4359a64d30f js/src/jit-test/tests/xdr/bug-1604747.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/xdr/bug-1604747.js	Fri Dec 20 16:38:35 2019 +0000
@@ -0,0 +1,9 @@
+let og = parseModule("1");
+let bc = codeModule(og);
+let m54 = decodeModule(bc);
+m54.declarationInstantiation();
+
+try {
+    bc = codeModule(m54);
+}
+catch {}